openpype/hosts/aftereffects/api/ws_stub.py

```diff
@@ -298,13 +298,14 @@ def import_file(self, path, item_name, import_options=None):
             self.client.call('AfterEffects.import_file',
                              path=path,
                              item_name=item_name,
-                             import_options=import_options)
+                             import_options=import_options,
```

Comment:
fps should be added in the `import_options`, not as a new argument of the function